assets/queries/terraform/aws/api_gateway_stage_without_api_gateway_usage_plan_associated/test/positive.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_api_gateway_stage" "positive1" {
+  rest_api_id   = "some deployment id"
+  deployment_id = "some rest api id"
+  stage_name = "some name"
+  tags {
+    project = "ProjectName"
+  }
+}
+
+resource "aws_api_gateway_stage" "positive2" {
+  deployment_id = "some deployment id"
+  rest_api_id   = "some rest api id"
+  stage_name    = "development"
+}
+
+resource "aws_api_gateway_usage_plan" "positive3" {
```

Comment:
Add another `aws_api_gateway_stage` that is actually connected to the `aws_api_gateway_usage_plan`.